### ISSUE TITLE
trivy k8s: unable to parse security checks

### ISSUE THREAD

**Issue by kanton10062006** [aquasecurity/trivy#2905]
Hi folks,

With recent release of trivy(0.32.0) we faced with unexpected behaviour of k8s scan:
```
trivy k8s all --report all -n staging -o trivy-ns-staging.txt --no-progress --exit-code 0 --timeout 60m
2022-09-19T10:43:58.709+0300	FATAL	flag error: scan flag error: unable to parse security checks: unknown security check: vuln,config,secret,rbac
```

trivy 0.31.3 version works fine with the same flags.

## Output of run with `-debug`:

```
trivy k8s all --report all -n staging -o trivy-ns-staging.txt --no-progress --exit-code 0 --timeout 60m --debug                                                                             
2022-09-19T11:04:05.139+0300	DEBUG	Severities: ["UNKNOWN" "LOW" "MEDIUM" "HIGH" "CRITICAL"]
2022-09-19T11:04:05.139+0300	FATAL	flag error:
    github.com/aquasecurity/trivy/pkg/commands.NewKubernetesCommand.func2
        /home/runner/work/trivy/trivy/pkg/commands/app.go:781
  - scan flag error:
    github.com/aquasecurity/trivy/pkg/flag.(*Flags).ToOptions
        /home/runner/work/trivy/trivy/pkg/flag/options.go:366
  - unable to parse security checks:
    github.com/aquasecurity/trivy/pkg/flag.(*ScanFlagGroup).ToOptions
        /home/runner/work/trivy/trivy/pkg/flag/scan_flags.go:103
  - unknown security check: vuln,config,secret,rbac:
    github.com/aquasecurity/trivy/pkg/flag.parseSecurityCheck
        /home/runner/work/trivy/trivy/pkg/flag/scan_flags.go:127
```

## Output of `trivy -v`:

```
trivy --version
Version: 0.32.0
Vulnerability DB:
  Version: 2
  UpdatedAt: 2022-09-16 12:09:04.028803627 +0000 UTC
  NextUpdate: 2022-09-16 18:09:04.028803427 +0000 UTC
  DownloadedAt: 2022-09-16 15:07:10.286422 +0000 UTC
```

Single '--security-checks vuln' scan flag works as expected.

Thanks!

**Comment by Strelox** [comment]
Same error with `trivy image` and passing multiple scan options via Environment Variable
```
TRIVY_SECURITY_CHECKS=vuln,config trivy image --clear-cache --debug
2022-09-19T12:26:09.271+0200	DEBUG	Severities: ["UNKNOWN" "LOW" "MEDIUM" "HIGH" "CRITICAL"]
2022-09-19T12:26:09.271+0200	FATAL	flag error:
    github.com/aquasecurity/trivy/pkg/commands.NewImageCommand.func2
        /home/runner/work/trivy/trivy/pkg/commands/app.go:267
  - scan flag error:
    github.com/aquasecurity/trivy/pkg/flag.(*Flags).ToOptions
        /home/runner/work/trivy/trivy/pkg/flag/options.go:366
  - unable to parse security checks:
    github.com/aquasecurity/trivy/pkg/flag.(*ScanFlagGroup).ToOptions
        /home/runner/work/trivy/trivy/pkg/flag/scan_flags.go:103
  - unknown security check: vuln,config:
    github.com/aquasecurity/trivy/pkg/flag.parseSecurityCheck
        /home/runner/work/trivy/trivy/pkg/flag/scan_flags.go:127
```

Passing options via `--security-checks vuln,config` works, though:
```
trivy i --security-checks vuln,config --clear-cache
2022-09-19T12:26:41.844+0200	INFO	Removing artifact caches...
```

Trivy Version: 0.32.0

**Comment by chen-keinan** [comment]
@kanton10062006 this issue has been fixed by #2926 and included with `trivy v0.32.1`

**Comment by patsevanton** [comment]
How correct cli command for  cluster all scanning ?
